Doc/tutorial/modules.rst

```diff
@@ -427,7 +427,7 @@ When importing the package, Python searches through the directories on
 
 The :file:`__init__.py` files are required to make Python treat directories
 containing the file as packages.  This prevents directories with a common name,
-such as ``string``, unintentionally hiding valid modules that occur later
+such as ``string``, unintentionally hide valid modules that occur later,
```

Comment:
In this case, both the original wording: 
```
This prevents directories with a common name,
such as ``string``, unintentionally hiding valid modules that occur later
on the module search path.
```

and the updated wording: 
```
This prevents directories with a common name,
such as ``string``, unintentionally hide valid modules that occur later,
on the module search path.
```

seem to be a bit grammatically incorrect. Instead, I would suggest the following:
```suggestion
such as ``string``, from unintentionally hiding valid modules that occur later
```

It seems be missing a "from". Changing "hiding" to "hide" and adding the comma between "later" and "on" would also be grammatically incorrect in this context though.

However, since you noticed there was an issue with this section, you can feel free to take the credit for the above correction if you'd like to. You can add it to the PR by using "Commit suggestion".